src/model.js

```diff
@@ -1059,6 +1061,9 @@ var Model = TransisObject.extend(function() {
         if (opts.except === name) { continue; }
         if (Array.isArray(opts.except) && opts.except.indexOf(name) >= 0) { continue; }
 
+        if (opts.only && opts.only !== name) { continue; }
```

Comment:
If `opts.only` is an array and `name` is in the array, then this will skip processing `name`.